### ISSUE TITLE
Support for fractional part

### ISSUE THREAD

**Issue by jtojnar** [fizzyizzy05/binary#22]
I recently needed to convert some decimal numbers with fractional part to binary, in the end having to rely on GNOME Calculator. It would be great if Binary could support it.

Possible issues:

- Exact terminating [conversion](https://en.wikipedia.org/wiki/Positional_notation#Base_conversion) is not always possible: 0.2₁₀ = 0.0̅0̅1̅1̅₂
- Different languages use different [fractional part separator](https://en.wikipedia.org/wiki/Decimal_separator) (e.g. 0,2 in Czech)

**Comment by fizzyizzy05** [comment]
Agreed, support for fractional parts would be good to have. I'll have a look at implementing this when I've got more spare time. I'm not entirely sure how to handle the separator for localisation, though it's probably easier to just have a list of different separators and check strings against it regardless of system language.
